load_organisms_and_genes_to_db.sh

```diff
@@ -0,0 +1,244 @@
+# Create organism records
+python manage.py organisms_create_or_update --taxonomy_id=9606 \
```

Comment:
Not sure whether it will be better to write a `for` loop to deal with these commands. What do you think?